src/ORM/Marshaller.php

```diff
@@ -77,6 +78,11 @@ protected function _buildPropertyMap($options)
             $assoc = $this->_table->association($key);
             if ($assoc) {
                 $map[$assoc->property()] = ['association' => $assoc] + $nested + ['associated' => []];
+            } else {
```

Comment:
i think you can flip this logic around to be without `else`

`if (!$assoc && substr($key, 0, 1) !== "_") {`